DigitalLearningSolutions.Web/Styles/shared/searchableElements/filter.scss

```diff
@@ -115,7 +120,7 @@
 
   #js-styling-hidden-area-while-loading & {
     .js-enabled & {
-      margin-right: 4px;
+      margin-right: 5px;

```

Comment:
Is this bit actually necessary if we get rid of the above

>   .js-enabled & {
    margin-right: 0;
  }

I'm wondering about potential consequences for when we have js-enabled, but we have disabled js styling when there are large numbers of items.

Reply:
Can you give me an example of when "disabled js styling" happens?

Reply:
It occurs when we have JS enabled, but the number of items on a page exceeds a certain limit. By default, it is 250 items. You can change that locally in the appsettings.Development.json file from the ` "JavascriptSearchSortFilterPaginateItemLimit": 250,` field.

There are some pages where this does not apply, but in the majority of cases (including Centre Course Setup) it does apply.

Reply:
Ok, I've looked at this scenario now, and my changes do not fix the issue when the JavascriptSearchSortFilterPaginateItemLimit is exceeded.

I have an alternative solution which appears to work, which is:
* Remove margin-right between active filter tags
* Add 5px margin-right to the `.applied-filter-container` (this change was part of the original commit)
* When adding applied filter tags in `filter.ts`, append a `document.createTextNode(' ')` after appending each filter

This latest bullet point makes the behaviour consistent whether js is enabled or not, i.e. there is a natural space between the tag elements always, rather than sometimes having a space and sometimes having a css margin.

I have checked the behaviour in the following circumstances, and found it to be correct in all cases (i.e. a space between each tag element, and a space between a long tag element and the clear filters button):
* JS enabled
* JS disabled
* JS enabled, but JavascriptSearchSortFilterPaginateItemLimit exceeded
* The above three cases, but with device dimension width = 400px (mobile)